tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial4.ipynb

```diff
@@ -829,9 +807,9 @@
    "name": "python3"
   },
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python [conda env:climatematch_2024] *",
    "language": "python",
-   "name": "python3"
+   "name": "conda-env-climatematch_2024-py"
   },
   "language_info": {
```

Comment:
We need to change this back to python3 before pushing

Reply:
True, sorry for that. Pushed update commit.